tests/app/helpers/test_template_helpers.py

```diff
@@ -246,12 +196,26 @@ def test_get_page_header_context(
             if cookie_value:
                 cookie_session[cookie_name] = cookie_value
 
-        result = ContextHelper(
-            language="en",
-            is_post_submission=False,
-            include_csrf_token=True,
-            survey_config=survey_config,
-        ).context["page_header"]
+        result = [
+            ContextHelper(
+                language="en",
+                is_post_submission=False,
+                include_csrf_token=True,
+                survey_config=survey_config,
+            ).context["survey_title"],
+            ContextHelper(
+                language="en",
+                is_post_submission=False,
+                include_csrf_token=True,
+                survey_config=survey_config,
+            ).context["mastheadLogo"],
+            ContextHelper(
+                language="en",
+                is_post_submission=False,
+                include_csrf_token=True,
+                survey_config=survey_config,
+            ).context["mastheadLogoMobile"],
+        ]
```

Comment:
Only need to call context helper once here 
```suggestion
              context_helper = ContextHelper(
            language="en",
            is_post_submission=False,
            include_csrf_token=True,
            survey_config=survey_config,
        )

        result = [
            context_helper.context["survey_title"],
            context_helper.context["mastheadLogo"],
            context_helper.context["mastheadLogoMobile"],
        ]
```